cmd/wallet-sdk-gomobile/api/credentials.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright Avast Software. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package api
```

Comment:
Might make more sense to put this in `openid4ci.go`, since it's actually based on the credential response type (not just a pure credential - more like a credential+format)